docs/source/contributing.rst

```diff
@@ -0,0 +1,3 @@
+.. highlight:: sh
```

Comment:
There's no chicken-egg-ness involved. The online HTML pages will show zero difference.
The only difference is if there is actual usable content in `CONTRIBUTING.rst`. And I'd prefer if there'd be.

I don't have a special workflow for when I'm offline.
When I want to run the tests, for example, I'd look in `CONTRIBUTING.rst` (which is available offline) for how to do that.
I could of course first build the HTML docs and look it up there, but that would be insanely inefficient. Also, if I had forgotten how to build the docs, I'd have to look it up in `CONTRIBUTING.rst` first ... but wait ... the information isn't there, I have to build the docs first ... but how? ...

And them I'm trapped in a loop until I reach my destination.